nextflow.config

```diff
@@ -13,6 +13,7 @@ params {
     input                                = null
     single_end                           = false
     assembly_input                       = null
+    nf_core_pipeline                     = null
```

Comment:
I'm not sure about this parameter name... I prefer things to be more active:

```
--generate_downstream_samplesheet
```

What about something like that?